demo/src/screens/componentScreens/SegmentedControlScreen.tsx

```diff
@@ -54,12 +54,12 @@ const SegmentedControlScreen = () => {
         </View>
         <SegmentedControl
           containerStyle={styles.container}
-          onChangeIndex={(index: number) => onChangeIndex('second', index)}
+          onChangeIndex={(index: number) => onChangeIndex('fifth', index)}
```

Comment:
Sometimes performance issue can be caused by the example screen. 
Consider not passing and inline anonymous callback

Reply:
I agree, but in this specific case, the performance issues are probably from the component itself since the complaint came from Dine devs.. 
Anyway, I removed the inline callbacks from the screen :)